app/src/main/res/layout/fraction_interaction_item.xml

```diff
@@ -10,16 +10,22 @@
       name="viewModel"
       type="org.oppia.android.app.player.state.itemviewmodel.FractionInteractionViewModel" />
   </data>
+  <androidx.constraintlayout.widget.ConstraintLayout
```

Comment:
Can we get rid of either the `ContraintLayout` or the `LinearLayout` ? Keeping both does not make sense.

Reply:
Done here and to other interactions too.